newsrevenuehub/rev-engine
# DEV-1608 NR Hub -> NR Engine

#### What's this PR do?
Update Hub to Engine
  - password reset form
  - password reset email
  - slack integration error log

Did not update all the "Hubs" in Slack integration as they are about News
Revenue Hub's Slack account. Where the integration email is about how
NRE is not configured to integrate that NRH slack account.

#### Why are we doing this? How does it help us?
Correctness

#### How should this be manually tested?
Navigate to password reset form (verify it says Engine, not Hub).
Request password reset email. (verify email says Engine, not Hub).
Not exactly sure about slack integration error log. It's an internal log line, I think it's ok to skip testing.

#### Have automated unit tests been added?
N/A

#### How should this change be communicated to end users?
N/A

#### Are there any smells or added technical debt to note?
No

#### Has this been documented? If so, where?
N/A

#### What are the relevant tickets?
[DEV-1608]

#### Do any changes need to be made before deployment to production (adding environment variables, for example)?
No

#### Are there next steps? If so, what? Have tickets been opened for them?
No

[DEV-1608]: https://news-revenue-hub.atlassian.net/browse/DEV-1608?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
